src/content/docs/new-relic-solutions/get-started/glossary.mdx

```diff
@@ -763,7 +847,9 @@ Whether you're considering New Relic One or you're already using our capabilitie
     id="metrics"
     title="metric"
   >
-    A **metric** is a numeric measurement. Metric data is a broad category because there are several ways to make and report measurements. For more about how metrics are reported at New Relic, see [New Relic data types](/docs/using-new-relic/data/understand-data/new-relic-data-types#metrics).
+  In the software monitoring industry, a metric means a numeric measurement of an application or system. 
+
+  To learn how New Relic uses metrics, see [New Relic data types](/docs/data-apis/understand-data/new-relic-data-types/). 
```

Comment:
Not sure if this will be too big a space. You might need to remove this extra line.

Reply:
Fixed this!